backend/src/packages/business/business.controller.ts

```diff
@@ -763,11 +767,17 @@ class BusinessController extends Controller {
   private async findAllDrivers(
     options: ApiHandlerOptions<{
       params: BusinessGetAllDriversRequestParameters;
+      query?: BusinessGetDriversPageRequestQuery;
     }>,
   ): Promise<ApiHandlerResponse> {
-    const drivers = await this.businessService.findAllDriversByBusinessId(
-      options.params.businessId,
-    );
+    const drivers = await (options.query
+      ? this.businessService.findPageOfDrivers({
+          ...options.params,
+          ...options.query,
+        })
+      : this.businessService.findAllDriversByBusinessId(
+          options.params.businessId,
+        ));
```

Comment:
do we actually need to find all business drivers without pagination? I don't think so
query params will be always present